### DISCUSSION TITLE
[FF8] hi-poly tex model replacement

### DISCUSSION THREAD

**Discussion by shikulja** [julianxhokaxhiu/FFNx#446] [Q&A] [answered]
Saw a great commit
https://github.com/julianxhokaxhiu/FFNx/commit/41b118b2c2e7363c0a4cdd550624ffeb40ff64df

The question arose, is it possible to do the same for 3d models? how it's done here:
https://forums.qhimm.com/index.php?topic=19979.0

Rationale. Replacing textures is not very practical with third party tools when the game contains everything you need. Moreover, distributing main_chr with ready-made imported hi-res textures is not very correct.

A simple LOD distance fix that improves the look of the game without adding HD textures.
There is probably a more elegant way.

**Comment by julianxhokaxhiu** [comment]
Are you speaking about replacing textures or models here? Because if textures, please see #29 as progress is being made. If you're speaking about models then #33 is the issue to follow.

Thank you in advance.

**Comment by shikulja** [comment]
@julianxhokaxhiu 
It's a little different. Replacing textures on models, yes, but using built-in hires textures that are used instead of lowres depending on the Lod of the game.
The description on the forum uses simple texture replacement. But perhaps there are other ways. Rewriting paths, or increasing the distance for Lod,
which will force the hires state of the textures for the models.

**Comment by julianxhokaxhiu** [comment, marked as answer]
On that regard we will use the same technique used on FF7 and you'll be able to do so, as soon as #29 is complete. I'll move this ticket to a discussion. Thank you.
